recipes/sdl/all/conanfile.py

```diff
@@ -310,6 +310,12 @@ def package_info(self):
             self.cpp_info.components["libsdl2"].system_libs = ["user32", "gdi32", "winmm", "imm32", "ole32", "oleaut32", "version", "uuid", "advapi32", "setupapi", "shell32"]
             if self.settings.compiler == "gcc":
                 self.cpp_info.components["libsdl2"].system_libs.append("mingw32")
+        elif self.settings.os == "Android":
+            self.cpp_info.components["libsdl2"].system_libs.extend(["android", "log"])
+            self.cpp_info.components["hidapi"].libs = ["hidapi"]
```

Comment:
Is it possible to use external lib instead?

Reply:
I will try at a later time.

Right now, I'm still at the stage of trying to get it building/working on Android.